src/@apollo/client/ApolloClient__ApolloClient.re

```diff
@@ -206,7 +207,7 @@ module ApolloClientOptions = {
       credentials: option(string),
       headers: option(Js.Dict.t(string)),
       link: option(ApolloLink.Js_.t),
-      cache: ApolloCache.Js_.t(Js.Json.t),
+      cache: ApolloCache.t(Js.Json.t), // Non-Js_ ApolloCache is correct here
```

Comment:
I'm sure there is a way to express `ApolloCache.t` is a subtype of `ApolloCache.Js_.t` but I don't have mental space or motivation to look into it at the moment.